tests/test_search.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import pytest
+
+from legistar import base
+
+
+class TestAPISearch(unittest.TestCase):
```

Comment:
Why use `unittest` here, instead of `pytest`?

Reply:
This is what pytest says to do? https://docs.pytest.org/en/latest/unittest.html

Reply:
It seems like that's what pytest says to do "for leveraging existing unittest-based test suites." Since `legistar` doesn't have an existing test suite (unless I missed it?), I think it'd be great to just use pytest conventions, with a fixture for setup in `conftest`, that's included in each test. You [can still use classes](https://docs.pytest.org/en/latest/getting-started.html#group-multiple-tests-in-a-class) to organize tests, if you like.

Reply:
@fgregg Is this something you want to do, in this PR?